mlflow/models/evaluation/base.py

```diff
@@ -657,78 +658,39 @@ def evaluate(
     evaluator_config=None,
 ):
     """
-    Evaluate a PyFunc model on the specified dataset using one or more specified evaluators, and
-    log resulting metrics & artifacts to MLflow Tracking.
-
-    :param model: A pyfunc model instance, or a URI referring to such a model.
-
-    :param data: One of the following:
-
-                  - A numpy array or list of evaluation features, excluding labels.
-
-                  - A Pandas DataFrame or Spark DataFrame, containing evaluation features and
-                    labels. If `feature_names` argument not specified, all columns are regarded
-                    as feature columns. Otherwise, only column names present in `feature_names`
-                    are regarded as feature columns.
+    Evaluate a PyFunc model on the specified dataset using one or more specified ``evaluators``, and
+    log resulting metrics & artifacts to MLflow Tracking. For additional overview information, see
+    :ref:`the Model Evaluation documentation <model-evaluation>`.
 
-    :param targets: If `data` is a numpy array or list, a numpy array or list of evaluation
-                    labels. If `data` is a DataFrame, the string name of a column from `data`
-                    that contains evaluation labels.
-
-    :param model_type: A string describing the model type. The default evaluator
-                       supports "regressor" and "classifier" as model types.
-
-    :param dataset_name: (Optional) The name of the dataset, must not contain double quotes (“).
-                         The name is logged to the `mlflow.datasets` tag for lineage tracking
-                         purposes. If not specified, the dataset hash is used as the dataset name.
-
-    :param dataset_path: (Optional) The path where the data is stored. Must not contain double
-                         quotes (“). If specified, the path is logged to the `mlflow.datasets`
-                         tag for lineage tracking purposes.
+    Default Evaluator behavior:
 
```

Comment:
Nit: Remove this blank line, so that the "Default Evaluator behavior:" header will displayed in bold format like:
![image](https://user-images.githubusercontent.com/19235986/149609498-7e73fe80-6c3f-4e7b-8787-7e4354146bdf.png)

Reply:
where is the blank line?

Reply:
Ah I see, line 688, right?

Reply:
https://github.com/mlflow/mlflow/pull/5272/commits/ed674879a413775e96386e2d307d0561efdcc248 fixed this.